1.11/release-notes/1.11.0-beta3/index.md

```diff
@@ -0,0 +1,86 @@
+
+<ul>
+<li>Upgrades from 1.10 are not supported.</li>
+<li>DC/OS 1.11 requires CLI version 0.6.x.
+  <ul>
+  <li><a href="/1.11/cli/uninstall/">Uninstall the existing CLI</a>.</li>
+  <li>Install version 0.6.x using the <strong>Install CLI</strong> instructions in the dropdown in the upper left hand corner of the 1.11 DC/OS GUI.</li>
+  </ul>
+<strong>Note:</strong> CLI version 0.6.x is not compatible with DC/OS 1.10</li>
+</ul>
+
+<a name="new-features"></a>
+# New features and capabilities
+
+## Apache Mesos 1.5 and Marathon 1.6 Integrated.
+- DC/OS 1.11 is is based on Mesos 1.5. View the [changelog](https://github.com/apache/mesos/blob/master/CHANGELOG).
+
+- DC/OS 1.11 is integrated with the latest release of Marathon, version 1.6. Resulting breaking changes and new features are documented below. For more information about Marathon 1.6, consult the [Marathon changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
+
+## Networking
+- IPv6 is now supported for Docker containers.
+  You can now launch docker containers through Marathon on an IPv6 Docker user network. Tthe built-in DC/OS overlay also supports IPv6 virtual networks for Docker containers. By default, DC/OS overlay now comes with an IPv6 virtual network called `dcos6`. You can use `dcos6` to launch IPv6 docker containers through Marathon.
+  **Note:** IPv6 is not supported for CNI, and hence UCR containers.
+- Edge-LB 1.0 RC candidate. [enterprise type="inline" size="small" /]
+  - Strict security mode is now supported.
+- Performance improvements to the DC/OS network stack.
+  All networking components (minuteman, navstar, spartan) are aggregated into a single Erlang VM called `dcos-net`. This improves the resource consumption of the Erlang components and greatly improves reliability and availability to these components.
+
+[enterprise]
+## Security
+[/enterprise]
+
+[enterprise]
+## Storage
+- DC/OS Storage Service now in beta. DC/OS 1.11 introduces a single, highly-available, control plane end point to manage and consume storage services and features within DC/OS.
+- A Container Storage Interface (CSI) that enables you to connect to locally managed and 3rd party storage features via a common pluggable model.
+- Volume profiles. You can now create a volume profile with your desired storage parameters, then apply it during volume creation. This simplifies and expedites volume creation.
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).
+[/enterprise]
+
+## Metrics
+- Metrics - The DC/OS metrics component now produces metrics in Prometheus format. [View the documentation](/1.11/metrics).
+
+## Platform
+- Fault domain awareness. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed. [View the documentation](/1.11/deploying-services/fault-domain-awareness). [enterprise type="inline" size="small" /]
```

Comment:
Both "Fault domain awareness" and "Linked clusters" should be wrapped in a enterprise block. /cc @somikbehera @keithchambers